# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libjar/zipwriter/nsZipWriter.cpp
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c modules/libjar/zipwriter/nsZipWriter.cpp
--- a/modules/libjar/zipwriter/nsZipWriter.cpp	Tue Feb 12 17:58:19 2019 +0100
+++ b/modules/libjar/zipwriter/nsZipWriter.cpp	Tue Feb 12 16:08:25 2019 +0000
@@ -376,7 +376,7 @@
 
   nsCOMPtr<nsIInputStream> inputStream;
   nsresult rv =
-      NS_MaybeOpenChannelUsingOpen2(aChannel, getter_AddRefs(inputStream));
+      NS_MaybeOpenChannelUsingOpen(aChannel, getter_AddRefs(inputStream));
 
   NS_ENSURE_SUCCESS(rv, rv);
 
@@ -915,7 +915,7 @@
       rv = pump->AsyncRead(stream, nullptr);
       NS_ENSURE_SUCCESS(rv, rv);
     } else {
-      rv = NS_MaybeOpenChannelUsingAsyncOpen2(aItem->mChannel, stream);
+      rv = NS_MaybeOpenChannelUsingAsyncOpen(aItem->mChannel, stream);
       NS_ENSURE_SUCCESS(rv, rv);
     }
 